src/KGzocha/Searcher/Context/Doctrine/CachedQueryBuilderSearchingContext.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the searcher package.
+ *
+ * (c) Daniel Ribeiro <drgomesp@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace KGzocha\Searcher\Context\Doctrine;
+
+/**
+ * Class CachedQueryBuilderSearchingContext
+ *
+ * @author Daniel Ribeiro <drgomesp@gmail.com>
+ * @package KGzocha\Searcher\Context\Doctrine
```

Comment:
Instead of `@package` could you add class description in this PHPDocs? One sentence is enough - just like other classes/services